common/src/main/java/com/microsoft/identity/common/internal/ui/AndroidAuthorizationStrategyFactory.java

```diff
@@ -54,69 +47,77 @@
 @SuppressWarnings(WarningType.rawtype_warning)
 @Builder
 @Accessors(prefix = "m")
-public class AndroidAuthorizationStrategyFactory implements IAuthorizationStrategyFactory{
+public class AndroidAuthorizationStrategyFactory implements IAuthorizationStrategyFactory<IAuthorizationStrategy> {
     private static final String TAG = AndroidAuthorizationStrategyFactory.class.getSimpleName();
 
     private final Context mContext;
     private final Activity mActivity;
     private final Fragment mFragment;
 
+    /**
+     * Get the authorization strategy.
+     *
+     * @param authorizationAgent The authorization agent provided by the caller.
+     * @param browser            The browser to use for authorization.
+     * @param isBrowserRequest   True if the request is from browser.
+     * @return The authorization strategy.
+     */
     @Override
+    @NonNull
     public IAuthorizationStrategy getAuthorizationStrategy(
-            @NonNull final InteractiveTokenCommandParameters parameters) {
+            @NonNull final AuthorizationAgent authorizationAgent,
+            @Nullable final Browser browser,
+            final boolean isBrowserRequest) {
```

Comment:
>isBrowserRequest [](http://example.com/codeflow?start=26&length=16)

This is isBrowserRequest here but is used for isBrokerRequest when calling getBrowserAuthorizationStrategy.

I guess it should be isBrokerRequest